services/user-management/src/main/java/com/foodie/user/controllers/PermissionController.java

```diff
@@ -18,9 +19,9 @@
 @Slf4j
 @RestController
 @CrossOrigin
-@RequestMapping("/api/v1/users/permissions")
+@RequestMapping("/api/v1/security/permissions")
```

Comment:
In Prod you should consider finding a way to make `v1` dynamic

Reply:
you mean api/v1 right

Reply:
Yes